src/Verifier.php

```diff
@@ -186,15 +184,13 @@ protected function verifyTrustedChecksumList($checksum_list_raw, $working_direct
                 throw new VerifierException("Failure computing hash for file \"$file_checksum->filename\" in the checksum list.");
             }
 
-            if (strcmp($actual_hash, $file_checksum->hex_hash) !== 0)
+            if ($actual_hash !== $file_checksum->hex_hash)
```

Comment:
I seem to recall a comment made at one point that `!==` was sufficient. So taking advantage of this PR to change that.

Reply:
`!==` is absolutely fine here.